MIGRATING.md

```diff
@@ -2,6 +2,94 @@
+- `ListAppLinks(String userId,Boolean showAll)`
+<br />&mdash; Signature changed `ListAppLinks(String userId)`
+- `ListUserGroups(String userId,String after,Int32 limit)`
+<br />&mdash; Signature changed `ListUserGroups(String userId)`
+- `ExpirePasswordAsync(String userId,Boolean tempPassword,CancellationToken cancellationToken)`
+<br />&mdash; Signature changed `ExpirePasswordAsync(String userId,CancellationToken cancellationToken)` 
+- `ResetAllFactorsAsync(String userId,CancellationToken cancellationToken)`
+<br />&mdash; Renamed `ResetFactorsAsync(String userId,CancellationToken cancellationToken)`
+- `ResetPasswordAsync(String userId,Okta.Sdk.AuthenticationProviderType provider,Boolean sendEmail,CancellationToken cancellationToken)`
+<br />&mdash; Removed; instead use any of the following:
+  - `ForgotPasswordGenerateOneTimeTokenAsync(String userId,Boolean sendEmail,CancellationToken cancellationToken)`
+  - `ForgotPasswordSetNewPasswordAsync(Okta.Sdk.IUserCredentials user,String userId,Boolean sendEmail,CancellationToken cancellationToken)`
+  - `ExpirePasswordAsync(String userId,CancellationToken cancellationToken)`
+  - `ExpirePasswordAndGetTemporaryPasswordAsync(String userId,CancellationToken cancellationToken)`
+- `ListAssignedRoles(String userId,String expand)`
+<br />&mdash; Renamed `ListAssignedRolesForUser(String userId,String expand)`
+- `EndAllUserSessionsAsync(String userId,Boolean oauthTokens,CancellationToken cancellationToken)`
+<br />&mdash; Renamed `ClearUserSessionsAsync(String userId,Boolean oauthTokens,CancellationToken cancellationToken)`
+
+### New Okta Clients
+The following is a list of context specific clients that are new in version 3.0.  Instances of each are available as properties of an OktaClient instance where the name of the property is the name of the type with the "Client" suffix removed.
+
+- `Okta.Sdk.AuthorizationServersClient`, see [Authorization Servers](https://developer.okta.com/docs/reference/api/authorization-servers/).
+- `Okta.Sdk.EventHooksClient`, see [Event Hooks Management API](https://developer.okta.com/docs/reference/api/event-hooks/).
+- `Okta.Sdk.FeaturesClient`, see [Features API](https://developer.okta.com/docs/reference/api/features/).
+- `Okta.Sdk.IdentityProvidersClient`, see [Identity Providers API](https://developer.okta.com/docs/reference/api/idps/).
+- `Okta.Sdk.InlineHooksClient`, see [Inline Hooks Management API](https://developer.okta.com/docs/reference/api/inline-hooks/).
+- `Okta.Sdk.LinkedObjectsClient`, see [Linked Objects API](https://developer.okta.com/docs/reference/api/linked-objects/).
+- `Okta.Sdk.TemplatesClient`, see [Custom Templates API](https://developer.okta.com/docs/reference/api/templates/).
+- `Okta.Sdk.TrustedOriginsClient`, see [Trusted Origins API](https://developer.okta.com/docs/reference/api/trusted-origins/).
```

Comment:
Question: Did we add a client for this? I don't remember seeing this in the spec.

Reply:
Looks like this was added in the previous round of changes.

Reply:
I see, but it is gonna be available in 3.0.0 right?

Reply:
Yes